client/ayon_hiero/api/pipeline.py

```diff
@@ -97,7 +141,7 @@ def containerise(track_item,
     """
 
     data_imprint = OrderedDict({
-        "schema": "openpype:container-2.0",
+        "schema": "ayon:container-3.0",
         "id": AVALON_CONTAINER_ID,
```

Comment:
With this schema change this value for `id` should be `ayon.load.container` which likely is `AYON_CONTAINER_ID`?
See: https://github.com/ynput/ayon-core/blob/ed39b769cfc47bf3e12cfe86987c37a018531d8d/client/ayon_core/pipeline/schema/container-3.0.json#L30

Reply:
Thanks for this !